python/ray/tune/examples/mnist_pytorch_trainable.py

```diff
@@ -45,6 +45,8 @@ def _setup(self, config):
             momentum=config.get("momentum", 0.9))
 
     def _train(self):
+        if random.random() < .1:
```

Comment:
@richardliaw what's the best way to do this, add a separate example script like `dashboard_demo.py`?

Reply:
I accidentally left these changes here, never meant to commit them. Just reverted all the examples back to the original version. If we want to keep an example to test different features of the dashboard, I can make a new one.